fluent/migrate/context.py

```diff
@@ -164,6 +149,22 @@ def get_sources(acc, cur):
                 acc.add((cur.path, cur.key))
             return acc
 
+        refpath = os.path.join(self.reference_dir, reference)
+        try:
+            ast = self.read_ftl_resource(refpath)
+        except IOError as err:
+            logger = logging.getLogger('migrate')
+            logger.error('Missing reference file: {}'.format(refpath))
+            sys.exit()
+        except UnicodeDecodeError as err:
+            logger = logging.getLogger('migrate')
+            logger.error('Error reading file {}: {}'.format(refpath, err))
+            sys.exit()
```

Comment:
Thanks, that's going to be cleaner indeed. I'll still use `sys.exit` because I don't have access to the `ArgumentParser` instance inside of `main`. Also `parser.exit` is just a wrapper around `sys.exit` anyways, intended AFAICT for errors about the argument setup.